src/lib/redis.js

```diff
@@ -0,0 +1,16 @@
+import Redis from 'redis'
+
+/**
+ * Return Redis Client
+ * @param {config} config 
+ */
+export function getClient(config) {
```

Comment:
Here we should check the same as I have wrote for the elasticsearch client.

Reply:
I'd preffer to have the client lifecycle managementt in the user scope